Assets/HoloToolkit/UI/Scripts/UsabilityUtilities.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// A helper class for making applications more usable across different devices.
+    /// </summary>
+    public static class UsabilityUtilities
```

Comment:
That's a good question, it might be that we default to 42 (~what HoloLens is), and people can adjust based on their experience.  I think sqrt(targetPPDOf42)/sqrt(cameraPPD) will get you a value that is close to what you've calculated through user testing (1.96 vs 2.18).